cmd/simpleargs/main.go

```diff
@@ -2,130 +2,112 @@ package main
 
 import (
 	"context"
-	"encoding/base64"
 	"fmt"
 	"io/ioutil"
 	"log"
-	"net/http"
 	"os"
 	"path/filepath"
 	"regexp"
 	"strconv"
 
-	"github.com/google/go-github/github"
+	"github.com/google/go-github/v26/github"
+	"golang.org/x/oauth2"
 	"gopkg.in/alecthomas/kingpin.v2"
 )
+const prombenchURL = "http://prombench.prometheus.io"
 
 var regex string
-var eventfile string
-var writepath string
+var input string
+var output string
+var owner string
+var repo string
+var prnumber int
+var releaseVersion string
```

Comment:
as I mention global variables is an anti patern and leads to problems at a later stage.